assets/scss/02-molecules/_listing-table.scss

```diff
@@ -70,6 +70,9 @@
 
   &__extra {
     display: none;
+    &.expanded {

```

Comment:
you also need to add a transition animation the the `__extra` block in order for the collapse to animate.
you should be able to use what is in accordion-items scss

```
&.collapsed {
  visibility: hidden;
  height: 0px;
}

&.collapsing, &.expanding {
  position: relative;
  overflow: hidden;
  visibility: visible;
  -webkit-transition-property: height, visibility;
  transition-property: height, visibility;
  transition-duration: 0.35s;
  -webkit-transition-duration: 0.35s;
  transition-timing-function: ease;
  -webkit-transition-timing-function: ease;
}

&.expanded {
  height: auto;
  visibility: visible;
  display: block;
}
```

Reply:
Updated